Products/Products-Root/Screens/Authentication/Register/RegisterViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  RegisterViewController.swift
+//  Products
+//
+//  Created by Dan Pop on 13/03/2020.
+//  Copyright © 2020 Archlime. All rights reserved.
+//
+
+import UIKit
+
+protocol RegisterViewControllerDelegate: class {
+    func testBtnPressed()
+}
+
+class RegisterViewController: BaseViewController {
+    
+    weak var delegate: RegisterViewControllerDelegate?
```

Comment:
please add a mark "Properties" here. Check the first answer here on how to create a MARK code snippet (https://stackoverflow.com/questions/44432645/is-there-any-shortcut-for-mark-in-xcode-like-there-is-for-description)